src/Components/Pickers/Internal/Picker.base.styles.ts

```diff
@@ -8,22 +8,34 @@ const classNames = {
 export const getStyles = (_props: IPickerBaseStyleProps): IPickerBaseStyles => {
     return {
         root: [classNames.root],
-        button: [
-            classNames.button,
-            {
-                border: '1px solid var(--cb-color-input-border)',
-                borderRadius: '50%',
-                cursor: 'pointer',
-                height: 28,
-                width: 28
-            }
-        ],
+        button: [],
         subComponentStyles: {
             callout: {
                 calloutMain: {
                     // this is not exposed on the styles directly
                     '& .ms-swatchColorPickerBodyContainer': {
-                        minWidth: 'unset'
+                        minWidth: 'unset',
+                        ' .ms-Button': {
+                            cursor: 'pointer',
+                            backgroundColor: 'transparent',
+                            width: 28,
+                            height: 28,
+                            margin: 0,
+                            padding: '2px',
+                            borderRadius: 0,
+                            borderStyle: 'solid',
+                            borderWidth: 0,
+                            ':hover': {
+                                backgroundColor:
+                                    'var(--cb-color-card-contrast)',
```

Comment:
do you think we can do better with on hover background in darker theme colors? In the screenshots it looks like more white bg color on hover, but generally it is more dark grey on hover of icon buttons in fluent design, can we make it here too? https://developer.microsoft.com/en-us/fluentui#/controls/web/button 
![image](https://user-images.githubusercontent.com/45217314/208156034-26ec24f5-0e54-4501-958f-eb2cc4cd7721.png)

Reply:
We definitely can. The color picker is mirroring the values used for icon picker so if we're changing the coloration, we should change in both. I'll chase this down with design.

Reply:
Moved to semantic color in theme as the light theme variable was the only value applying.